schema.sql

```diff
@@ -106,3 +106,7 @@ ON DELETE CASCADE;
 
 ALTER TABLE visits
 ADD date_of_visits DATE;
+
```

Comment:
Great job team 

- [ ] Kindly ensure the `schema.sql` contains indexes for FOREIGN KEYS or on columns frequently used in queries which is one of the basic solutions for the performance problems.
   CREATE INDEX ON visits (animal_id);
   CREATE INDEX ON visits (vet_id);
   CREATE INDEX ON owners (email);